package.json

```diff
@@ -9,7 +9,7 @@
     "async": "^3.2.0",
     "body-parser": "~1.19.0",
     "bytes": "^3.1.0",
-    "cloudinary": "^1.28.1",
+    "cloudinary": "https://github.com/cloudinary/cloudinary_npm.git#analyze_api_legacy",
```

Comment:
I suggest using a tag of the repository and not a branch because people might commit to that branch